### DISCUSSION TITLE
ImportError: cannot import name 'init_std_stream_encoding' from 'conda.common.compat'

### DISCUSSION THREAD

**Discussion by NadiaTamayo15** [biostars/biostar-handbook-issues#214] [Q&A] [answered]
Hi, I am pm the 9nth step of the installation and I have this problem I do not know how to solve. If anyone can help me I will appreciate it so much. Thank you 
  File "/home/nadiat/miniconda3/condabin/mamba", line 11, in <module>
    sys.exit(main())
  File "/home/nadiat/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 892, in main
    from conda.common.compat import ensure_text_type, init_std_stream_encoding
ImportError: cannot import name 'init_std_stream_encoding' from 'conda.common.compat' (/home/nadiat/miniconda3/lib/python3.9/site-packages/conda/common/compat.py)

I am currently using PopOS!

**Comment by ialbert** [comment]
Type `conda` instead of `mamba`, does the error persists? `conda` and `mamba` should be interchangeable, the latter is faster

**Reply by ialbert** [reply]
in general it sounds like something went bad overall, some commands were not followed properly, 

delete the miniconda3 directory with:

    rm -rf ~/miniconda3

then repeat the installation process.

**Reply by NadiaTamayo15** [reply]
I removed the directory and start the installation again m}but now on the third step I got this output. 

$ ~/miniconda3/condabin/conda init bash
no change     /home/nadiat/miniconda3/condabin/conda
no change     /home/nadiat/miniconda3/bin/conda
no change     /home/nadiat/miniconda3/bin/conda-env
no change     /home/nadiat/miniconda3/bin/activate
no change     /home/nadiat/miniconda3/bin/deactivate
no change     /home/nadiat/miniconda3/etc/profile.d/conda.sh
no change     /home/nadiat/miniconda3/etc/fish/conf.d/conda.fish
no change     /home/nadiat/miniconda3/shell/condabin/Conda.psm1
no change     /home/nadiat/miniconda3/shell/condabin/conda-hook.ps1
no change     /home/nadiat/miniconda3/lib/python3.9/site-packages/xontrib/conda.xsh
no change     /home/nadiat/miniconda3/etc/profile.d/conda.csh
no change     /home/nadiat/.bashrc
No action taken.

**Reply by NadiaTamayo15** [reply]
I think I know what the problem was I tried to install it at first with the automated installation and I did not check if I already had the bash_profile . Now I have 2 bash profiles. How can I fix this problem? I am so sorry 
nadiat@pop-os ~
$ grep BIOSTAR_HANDBOOK_START ~/.bashrc
# >>> BIOSTAR_HANDBOOK_START
# >>> BIOSTAR_HANDBOOK_START

**Comment by ialbert** [comment, marked as answer]
you can edit the file

    pico ~/.bashrc

delete all the commands in the second block of >>> START to >>> END

then press ctrl-O to save the file and ctrl-X to exit.

**Reply by NadiaTamayo15** [reply]
Thank you so much
